sdk/src/ingredient.rs

```diff
@@ -544,15 +544,15 @@ impl Ingredient {
         }
     }
 
-    /// Generates an `Ingredient` from a stream, including XMP info
+    /// Generates an `Ingredient` from a stream, including XMP info.
     pub fn from_stream_info<F, S>(stream: &mut dyn CAIRead, format: F, title: S) -> Self
     where
         F: Into<String>,
         S: Into<String>,
     {
         let format = format.into();
 
-        // try to get xmp info, if this fails all XmpInfo fields will be None
+        // try to get xmp info, if this fails all XmpInfo fields will be None.
```

Comment:
Capitalize the T at the beginning.

Reply:
Addressed, although this is not a doc comment.